src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -249,3 +257,19 @@ export default {
 	}
 };
 </script>
+
+<style lang="postcss">
```

Comment:
Instead of unscoped CSS, you can use `scoped` on the `style` tag and `>>>` below, for example:

`#customizedSortBySelector >>> select {`

Reply:
Nice suggestion! I'll update it.